nats-ballerina/build.gradle

```diff
@@ -1,295 +1,295 @@
+//        }
+//
+//        copy {
+//            from targetBallerinaJar
+//            into file("$artifactLibParent/libs")
+//        }
+//
+//        copy {
+//            from targetNativeJar
+//            into file("$artifactLibParent/libs")
+//        }
+//
+//        copy {
+//            from configurations.externalJars
+//            into file("$artifactLibParent/libs")
+//        }
+//
+//        // Publish to central
+//        if (!project.version.endsWith('-SNAPSHOT') && ballerinaCentralAccessToken != null && project.hasProperty("publishToCentral")) {
+//            println("Publishing to the ballerina central..")
+//            exec {
+//                workingDir project.projectDir
+//                environment "JAVA_OPTS", "-DBALLERINA_DEV_COMPILE_BALLERINA_ORG=true"
+//                if (Os.isFamily(Os.FAMILY_WINDOWS)) {
+//                    commandLine 'cmd', '/c', "$distributionBinPath/ballerina.bat push ${moduleName}  && exit %%ERRORLEVEL%%"
+//                } else {
+//                    commandLine 'sh', '-c', "$distributionBinPath/ballerina push ${moduleName}"
+//                }
+//            }
+//
+//        }
+//        // Doc creation and packing
+//        exec {
+//            workingDir project.projectDir
+//            environment "JAVA_OPTS", "-DBALLERINA_DEV_COMPILE_BALLERINA_ORG=true"
+//            if (Os.isFamily(Os.FAMILY_WINDOWS)) {
+//                commandLine 'cmd', '/c', "$distributionBinPath/ballerina.bat doc ${moduleName} && exit %%ERRORLEVEL%%"
+//            } else {
+//                commandLine 'sh', '-c', "$distributionBinPath/ballerina doc ${moduleName}"
+//            }
+//        }
+//        copy {
+//            from file("$project.projectDir/target/apidocs/${moduleName}")
+//            into file("$project.projectDir/build/docs_parent/docs/${moduleName}")
+//        }
+//    }
+//
+//    outputs.dir artifactCacheParent
+//    outputs.dir artifactBallerinaDocs
+//    outputs.dir artifactLibParent
+//}
+//
+//task createArtifactZip(type: Zip) {
+//    destinationDirectory = file("$buildDir/distributions")
+//    from ballerinaBuild
+//}
+//
+//publishing {
+//    publications {
+//        mavenJava(MavenPublication) {
+//            artifact source: createArtifactZip, extension: 'zip'
+//        }
+//    }
+//
+//    repositories {
+//        maven {
+//            name = "GitHubPackages"
+//            url = uri("https://maven.pkg.github.com/ballerina-platform/module-ballerinax-nats")
+//            credentials {
+//                username = System.getenv("packageUser")
+//                password = System.getenv("packagePAT")
+//            }
+//        }
+//    }
+//}
+//
+//unpackJballerinaTools.dependsOn copyToLib
+//unpackStdLibs.dependsOn unpackJballerinaTools
+//copyStdlibs.dependsOn unpackStdLibs
+//updateTomlVersions.dependsOn copyStdlibs
+//ballerinaBuild.dependsOn updateTomlVersions
+//ballerinaBuild.dependsOn ":nats-native:build"
+//build.dependsOn ballerinaBuild
```

Comment:
why are we commenting this `build.gradle` file.

Reply:
Fixed.